ballerina/api_inspectionlot_srv/Package.md

```diff
@@ -70,7 +70,7 @@ inspectionlot:Client inspectionlotClient = check new (
 Now, utilize the available connector operations.
 
 ```ballerina
-inspectionlot:Wrapper createdInspectionlot = check inspectionlotClient->createA_Inspectionlot();
+inspectionlot:A_InspectionLotWrapper createdInspectionlot = check inspectionlotClient->createA_Inspectionlot();
```

Comment:
Doesnt the create require payload? Wont list be better?

Reply:
Updated here [Update docs.json in both modules](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/12/commits/c21a6a16b9b47fb17256167ac2be9a83b289f8c6)